CDP4Composition/CDP4Composition_avktpin5_wpftmp.csproj

```diff
@@ -1,1572 +0,0 @@
-<Project Sdk="Microsoft.NET.Sdk">
```

Comment:
this file should probably not be here

Reply:
True. You're seeing this because it was deleted in this commit. So this is outdated.